strabo/private_helper.py

```diff
@@ -24,14 +24,18 @@ def fill_interest_point(ip,image_ids,form_title,form_body,form_geo_obj,form_laye
     ip.icon = form_icon
     ip.images = [db.session.query(schema.Images).get(int(id)) for id in image_ids]
 
-def fill_image(image,form_file_obj,form_descrip,year,month,day):
+def fill_image(image,form_file_obj,form_descrip,year,month):
     '''
-    if a flask.files object is passed in, then  
+    If a flask.files object is passed in, then it saves the new file and stores the new information in the database.
```

Comment:
I added a link to clear this up in the docstring. http://flask.pocoo.org/docs/0.11/patterns/fileuploads/.

Basically it is what the client sends to the server when you upload a file, and it allows you to save that file somewhere.